app/Models/User.php

```diff
@@ -561,6 +561,18 @@ public function two_factor_active()
         return false;
     }
 
+    /**
+     * Get action logs history for this asset
+     *
+     * @author [A. Gianotto] [<snipe@snipe.net>]
+     * @since [v1.0]
+     * @return \Illuminate\Database\Eloquent\Relations\Relation
+     */
+    public function adminuser()
```

Comment:
Could we call this instead `createdBy()`? I prefer the db fieldname to line up pretty closely to the methodname